build/pkgs/fricas/distros/gentoo.txt

```diff
@@ -0,0 +1 @@
+sci-mathematics/fricas[sbcl]
```

Comment:
Why do you impose it to be build with `sbcl`. Sure it is probably the fastest lisp around, but is there a specific reason that you need to have it?

Reply:
ok